GitHubContents.ps1

```diff
@@ -358,7 +385,14 @@ filter Set-GitHubContent
 
     $uriFragment = "/repos/$OwnerName/$RepositoryName/contents/$Path"
 
-    $encodedContent = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($Content))
+    if ($ContentPath)
```

Comment:
I generally don't link to do boolean checks on strings.  This should either be something like:

```suggestion
    if (-not [String]::IsNullOrWhitespace($ContentPath))
```

or

```suggestion
    if (-not [String]::IsNullOrWhitespace($ContentPath))
```

or

```suggestion
    if ($PSCmdlet.ParameterSetName -in @('ElementsContent', 'UriContent'))
```

I think the **second one** is the most clear on the expectation that we should only go into this branch _if this parameter was provided at all_.